### DISCUSSION TITLE
Physics engine support?

### DISCUSSION THREAD

**Discussion by 19lmyers** [littlektframework/littlekt#213] [General]
Hello,
I've been trying to figure out if there exists a multiplatform-ready physics engine that could be integrated with LittleKt.

I know of Box2D, but the Java port isn't multiplatform (and hasn't been updated on Maven Central for quite some time). 
I also know KorGE maintains a port of Box2D, but it's impossible to pull it in without including KorGE itself (which I'd rather not do).

Has anyone managed to successfully use a physics engine with LittleKt?

**Comment by LeHaine** [comment]
Not that I am aware of. I did know of the Box2D port that KorGE maintains but didn't realize it pulled in all of KorGE. 

Perhaps we could take that port and pull whatever KorGE specific items out so we can have an implementation that doesn't have any large dependencies like that, if any at all.
